Support: add note about compiler versions (#1708)
Semmle/ql: change-notes/support/README.md

# Files moved to ``docs`` directory

> Now that all the QL documentation is in the `Semmle/ql` repository,

Reviewer: ```suggestion
Now that all of the QL documentation is in this repository,
```
Suggesting this change, because to me using the full name of the repo you are already in sounds a little bit strange.

> notes on the languages, compilers and frameworks supported have moved.

Reviewer: ```suggestion
notes on the languages, compilers, and frameworks supported have moved.
```

> They're now stored as part of the Sphinx support project with the other documentation:

Reviewer: ```suggestion
They're now stored as part of the Sphinx ``support`` project with the other documentation:
```
I think the project name is easier to identify if it's mono-spaced.
